config/routes.rb

```diff
@@ -1,6 +1,20 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  # root "users#index"
+  unauthenticated :user do
+    root 'splash#index', as: :unauthenticated_root
+  end
+
+  authenticated :user do
+    root 'categories#index', as: :authenticated_root
+  end
+  devise_for :users
+  resources :users
+  resources :categories do
+    resources :products
+  end
```

Comment:
- [ ] Kindly only expose the routes you are using, in this case, `create, index, new` to be used.

Reply:
Hi @elisha2kyakpo1, thanks for the review